app/models/training_data_measurement.rb

```diff
@@ -0,0 +1,2 @@
+class TrainingDataMeasurement < ApplicationRecord
```

Comment:
Emtpy model. Please define default values for all fields.

Reply:
I forgot to put the default values. Sorry

Reply:
Don't be sorry :+1: :)